lib/workers/repository/init/merge.ts

```diff
@@ -214,6 +214,18 @@ export async function mergeRenovateConfig(
     logger.debug('Found npmrc in decrypted config - setting');
     npmApi.setNpmrc(decryptedConfig.npmrc);
   }
+
+  const shallowResolvedConfig = await decryptConfig(
+    await presets.resolveConfigPresets(
+      decryptedConfig,
+      config,
+      [],
+      [],
+      true // resolve shallow config
+    ),
+    repository
+  );
+  logger.debug({ config: shallowResolvedConfig }, 'shallow config');
```

Comment:
shouldn't this conditional on a config options, as it can be expensive to resolve a config twice?

Reply:
ok